examples/on_gke_end_to_end/main.tf

```diff
@@ -152,20 +154,30 @@ module "forseti-on-gke" {
     helm       = "helm.forseti"
   }
   source                             = "../../modules/on_gke"
+  config_validator_enabled           = "${var.config_validator_enabled}"
   forseti_client_service_account     = "${module.forseti.forseti-client-service-account}"
   forseti_client_vm_ip               = "${module.forseti.forseti-client-vm-ip}"
   forseti_cloudsql_connection_name   = "${module.forseti.forseti-cloudsql-connection-name}"
   forseti_server_service_account     = "${module.forseti.forseti-server-service-account}"
   forseti_server_bucket              = "${module.forseti.forseti-server-storage-bucket}"
+  git_sync_image                     = "${var.git_sync_image}"
```

Comment:
There are a lot of variables needed for this git sync functionality. Is there a way for us to abstract this out?

Reply:
I agree, there are a lot of variables for this feature.  As we sub-modularize the module, I will endeavor to clean this up.

Reply:
Sounds good- thanks Ken!